### DISCUSSION TITLE
What will be the domain renewal price for a transferred domain?

### DISCUSSION THREAD

**Discussion by KaBankz** [vercel/community#2259] [Help] [answered]
### Summary

I currently have a domain that I bought from a separate registrar, but I am considering transferring it to Vercel. But I can't seem to find any indication of what the renewal price will be for the domain if transferred. I want to know if I will be paying more or less with Vercel as opposed to my current registrar before transferring.

**Comment by Pieparker** [comment, marked as answer]
Hey there! Depending on the gTLD and whether a domain is classified as "premium", this can vary.

For most .com domains, at the time of writing, our renewal price is $20.
